deploy.sample.php

```diff
@@ -45,6 +43,16 @@
     'var/versions',
     'var/sessions'
 ]);
+// the configuration files of pimcore that will be processed at creation
+set('pimcore_shared_configurations', [
+    'var/config/website-settings.php',
+    'var/config/reports.php',
+    'var/config/extensions.php',
```

Comment:
can you please remove the extensions.php? this one should come from the repository.